unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/ExploreV2/Scripts/Sections/PlacesAndEventsSection/SubSections/PlacesSubSection/FavoritesSubSection/IFavoritesSubSectionComponentView.cs

```diff
@@ -0,0 +1,82 @@
+using System;
+using System.Collections.Generic;
+using UnityEngine;
+
+public interface IFavoritesSubSectionComponentView : IPlacesAndEventsSubSectionComponentView
```

Comment:
Don't see clients of this interface. Please, create interfaces only when there is an external client which uses it (tests are considered to also be the client of the code). Remove it or create test-cases with mocks, that will use it